ui/src/app/utils/stores.tsx

```diff
@@ -128,6 +128,14 @@ export interface RuntimeStore {
 
 export const runtimeStore = atom<RuntimeStore>({workspaceNamespace: undefined, runtime: undefined});
 
+// runtime store states: undefined(initial state) -> Runtime (user selected) <--> null (delete only - no recreate)
```

Comment:
nit not runtime. Also, I think it make sense to just add persistentDisk into runtimeStore, instead of creating the new one

Reply:
That makes sense for the attached PD. The reason to separate them is for the unattached PD. They don't belong to a runtime. But we can do it for simplicity.